### PR TITLE
[MM-37897] - added fading to the "new channel category dropzone box"

### PR DESCRIPTION
**Pull request by michelengelen**
#### Summary
this PR adds a fading effect on the "new channel category dropzone box" for a better draggable drop indication.
I will add a short clip of the new fade to this PR

#### Ticket Link
[MM-37897](https://mattermost.atlassian.net/browse/MM-37897)

#### Screenshots
https://user-images.githubusercontent.com/32863416/131969038-3a645296-4d32-407e-ba65-5cc4176f8202.mov

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by matthewbirtch — APPROVED** [review]
Nice @michelengelen!

File: components/sidebar/sidebar_category/sidebar_category.tsx [hunk 1]
```diff
@@ -201,7 +201,7 @@ export default class SidebarCategory extends React.PureComponent<Props, State> {
                 <div className='SidebarCategory_newDropBox'>
                     <div
                         ref={this.newDropBoxRef}
-                        className={classNames('SidebarCategory_newDropBox-content', {
+                        className={classNames('SidebarCategory_newDropBox-content animating', {
```
Comment (deanwhillier) [inline components/sidebar/sidebar_category/sidebar_category.tsx:204]: `animating` as a verb implies that something is currently animating ... wonder if we should change to `animates` or `animated` to indicate that the element can/will animate ... or maybe no additional class is really needed at all?

If you do actually want to use `animating` as the class name, maybe only add it when the animation needs to happen and then remove once done as a number of popular libraries do for this kind of thing.

**Review by deanwhillier — CHANGES_REQUESTED** [review]
LGTM with one consideration.

File: components/sidebar/sidebar_category/sidebar_category.tsx (hunk 1 above)
Comment (michelengelen) [reply components/sidebar/sidebar_category/sidebar_category.tsx]: the `animating` was included even before I did my changes. I simple added it .. Although I just checked (and removed the hard coded class): It is actually added and removed automatically and I just did not notice that beforehand. 

Thanks for pointing that out! 👍🏼

Comment (deanwhillier) [reply components/sidebar/sidebar_category/sidebar_category.tsx]: Ah, lol, sounds good! 😄

**Review by deanwhillier — APPROVED** [review]

**Comment by michelengelen** [comment]
@nevyangelova and @jgilliam17 kind reminder to review

**Review by nevyangelova — APPROVED** [review]

**Comment by jgilliam17** [comment]
@michelengelen Test server is showing the "roles" error can you please take a look? Thanks

**Comment by michelengelen** [comment]
@jgilliam17 I fixed the issue for now, but we definitely need to look into this. A lot of the requests are failing when you enter the server and you have no active session.

@spirosoik and @hmhealey this is happening within redux ... i couldn't really figure out the root cause for this (IMO the failing requests are a big indicator :) ), but why are the APIs returning a HTTP status 401 in the first place? The selector `getCurrentUser` relies on the `state.entities.users.profile` object to be filled, but when there is no userProfile to be found it just returns `undefined`, which then breaks the application.

Maybe it fails at the `getCurrentUserId` selector already. I could not check, since it is working fine locally.

Attached is a screenshot of the console with the 401 errors:
Screenshot 2021-09-06 at 16 32 40

**Review by jgilliam17 — APPROVED** [review]
Thanks @michelengelen 
Tested, looks good to merge.
- Verified fading of the drag indicator when dropping a channel into new category.

**Pull request merged**